scss/styles.css

```diff
@@ -35,6 +35,80 @@
      object-fit: contain;
 }
 
+.sidebar {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  top: 0;
+  right: 0;
+  overflow: hidden;
+  transition: 0.5s;
+  padding-top: 3.75rem;
+  z-index: 1;
+  background-color: #FFFFFF;
+}
+
+.baritem {
+  padding: 0.5rem 1.8rem 0.5rem 1.8rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 1rem;
+  cursor: pointer;
+  text-align: left;
+}
+.baritem a {
+  text-decoration: none;
+  color: #000000;
+}
+
+a:first-child {
+  color: #fc7f3a;
+}
+
+a:hover {
+  color: #fc7f3a;
+}
+
+.sidebar div {
+  cursor: pointer;
+  text-align: left;
+}
+
+.sidebar .closebtn {
+  position: absolute;
+  top: 0;
+  left: 1.56rem;
+  font-size: 36px;
+  float: left;
+}
+
+.closebtn:hover {
+  color: #fc7f3a;
+}
+
+.openbtn:hover {
+  color: #fc7f3a;
+}
+
+.openbtn {
+  display: block;
+  font-size: 1.7rem;
+  cursor: pointer;
+  float: right;
+  background-color: #FFFFFF;
```

Comment:
and here

Reply:
![Screenshot 2023-10-17 075714](https://github.com/jroycodes/healthCareWebsite/assets/97290886/f6748736-9cd0-419c-bcc7-5bfc01043d19)